api/src/test/unit/java/uk/gov/dvsa/mot/vehicle/api/DateFormatAdapterTest.java

```diff
@@ -17,7 +17,7 @@
     private DateFormatAdapter adapter;
 
     {
-        theDate = Date.from(ZonedDateTime.of(2012, 3, 1, 4, 2, 4, 0, ZoneId.systemDefault()).toInstant());
```

Comment:
previously, unfortunately, this test was dependent on external resources.  This made him more integration test than unit ;-)